test/test_jit_cuda_fuser.py

```diff
@@ -4388,11 +4387,19 @@ def fn(x, y):
             torch._C._jit_nvfuser_clear_comparison_callback()
 
 class TestPassManagerCudaFuser(JitTestCase):
+    def setUp(self):
```

Comment:
Looks like we have some other test that leaked the flag and accidentally left nvfuser as enabled, or is it some thread racing that other tests running in parallel are turning the nvfuser switch?

Reply:
another test leaked the flag (my fault, fixed it on line 159)